governance/program/src/state/governance.rs

```diff
@@ -22,20 +24,30 @@ pub struct GovernanceConfig {
     /// Account governed by this Governance. It can be for example Program account, Mint account or Token Account
     pub governed_account: Pubkey,
 
-    /// Voting threshold of Yes votes in % required to tip the vote
-    /// It's the percentage of tokens out of the entire pool of governance tokens eligible to vote
-    // Note: If the threshold is below or equal to 50% then an even split of votes ex: 50:50 or 40:40 is always resolved as Defeated
-    // In other words +1 vote tie breaker is required to have successful vote
-    pub yes_vote_threshold_percentage: u8,
+    /// The type of the vote threshold used for voting
+    /// Note: In the current version only YesVote threshold is supported
+    pub vote_threshold_percentage_type: VoteThresholdPercentageType,
+
+    /// The vote threshold value of the type defined by vote_threshold_percentage_type
+    pub vote_threshold_percentage: u8,
```

Comment:
tiny little nit-idea: you could have `VoteThresholdType` be an enum-with-value, like `VoteThresholdType::YesVote(u8)`, since these two fields seem strongly intertwined.

Reply:
Cool idea, thx. I keep forgetting about this cool enum feature in Rust.